index.html

```diff
@@ -40,7 +40,176 @@ <h4>I'am a Software Developer</h4>
       </nav>
     </section>  
     <div class="bgimg"></div>
-    
+
+      <div class="recent">
+        <div class="title">
+          <h4>My Recent Works</h4>
+        </div>
+        <div class="divisor"></div>
+        <img id="placehold" src="./Images/Img Placeholder.png" alt="placeholder">
+      </div>
+        <div class="last">
+          <div id="subtitle">
+            <h5>Multi-Post Stories</h5>
+          </div>
+          <div id="description">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          </div>
+          <div id="tools">
+            <ul class="media">
+              <li><div class="cont1">css</div></li>
+              <li><div class="cont1">html</div></li>
+              <li><div class="cont1">bootstrap</div></li>
+              <li><div class="cont1">Ruby</div></li>
+            </ul>
+          </div>
+          <div>
+            <nav>
+              <button id="button1" type="button">See Project</button>
+            </nav>
+          </div>
+        </div>  
+      
+    <section class="works">
+      <div class="rest">
+        <div class="pj1">
+          <h3 class="tproj">Profesional Art Printing Data</h3>
+          <p class="desproj">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="media2">
+            <li><div class="cont1">html</div></li>
+            <li><div class="cont1">bootstrap</div></li>
+            <li><div class="cont1">Ruby</div></li>
+          </ul>
+          <nav id="contbut">
+            <button class="bigbutton" class="button2" type="button">See Project</button>
+          </nav>
```

Comment:
- [ ] Please don't wrap your buttons with the `<nav>` tag because It is representing the navigation bar of the page. It is usually one on the page. Please remove it. Instead, you can use the usual `div` tag for styling purposes😉